.github/DISCUSSION_TEMPLATE/Ideas.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+## Is your feature request related to a problem? Please describe.
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+## Describe the solution you'd like
+<!-- A clear and concise description of what you want to happen. -->
+
+## Describe alternatives you've considered
+<!-- A clear and concise description of any alternative solutions or features you've considered. -->
```

Comment:
Does it say about currently available alternatives in our project or alternative ideas?
Probably we need to ask about currently available alternatives in our project, if something is present

Reply:
changed the wording in here - to the one I think would better match the tone of "Ideas"